blocks/data-cards/data-cards.css

```diff
@@ -76,6 +76,10 @@
   display: flex;
 }
 
+.data-cards.news-card-variation .rte-container > div:nth-child(2) > div > p {
+  margin-block-end: var(--spacing-8);
```

Comment:
Don't use spacing variables. We are trying to get rid of it.

Reply:
Removed those variables for my changes and for the variables that are not available, I have given direct values.